.jshintrc

```diff
@@ -1,6 +1,4 @@
 {
-  "camelcase": true,
-  "curly": true,
```

Comment:
You got it. 
5014bc577515e22e78caa7804e341ea89207a3de
a4a4b4ce9da2d2fa07c918ef8f1b1f9b287916fb